packages/ui/src/theme/createTheme.ts

```diff
@@ -43,16 +43,7 @@ function setupTokens(obj: any, path = []) {
     for (const name in obj) {
       if (obj.hasOwnProperty(name)) {
         if (typeof obj[name] !== 'object') {
-          // If we get to this point that means there is a 'dangling' part of the theme object
-          // basically some part of the theme object that is not a design token, which is
-          // anything that is not an object with a value attribute
-          console.warn(
-            `Non-design token found when creating the theme at path: ${path.join(
-              '.'
-            )}.${name}\nDid you forget to add '{value:"${obj[name]}"}'?`
-          );
-          // Keep the users data there just in case
-          tokens[name] = obj[name];
+          tokens[name] = setupTokens({ value: obj[name] }, path.concat(name));
```

Comment:
Basically this is the change to allow custom values not contained in objects. If `obj[name]` is not an `object` then we assume that it should be assigned to a design token and continue the recursion